asset-transfer-basic/chaincode-java/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Basic asset transfer 
+
+This project demonstrates the use of the Java SDK, running a basic asset transfer contract using the "chaincode as a service" 
+pattern.
+
+
+## Setup 
+
+In this sample we will employ the [Kubernetes Test Network](../../test-network-k8s) to illustrate a scenario of 
+building, running, and debugging chaincode on a development workstation.  
+
+This project is also compatible with the legacy chaincode builder pipeline and the compose based test-network.
+For additional details, see the [End-to-end with the test-network](../../test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md#end-to-end-with-the-the-test-network)
+documentation.
+
+## [Quickstart](../../test-network-k8s#quickstart)
+
+```shell
+export PATH=${PWD}/../../test-network-k8s:$PATH 
+
+network kind 
+
+network up 
+network channel create
+```
+
+```shell
+network chaincode deploy  ${PWD}
+
+network chaincode invoke  asset-transfer-basic '{"Args":["InitLedger"]}'
+network chaincode query   asset-transfer-basic '{"Args":["ReadAsset","asset1"]}' | jq 
```

Comment:
I think we need `CreateAsset` chaincode first, don't we ?

Reply:
`InitLedger` will create a few assets from within a method in the chaincode.

CreateAsset is also OK to use.  It is just a little more "typing" of JSON, which is sometimes hard for my fingers.